dotcom-rendering/src/components/Branding.importable.tsx

```diff
@@ -24,17 +24,17 @@ function brandingLabelStyle(palette: Palette, format: ArticleFormat) {
 			return [
 				invariantStyles,
 				css`
-					color: ${neutral[20]};
+					color: ${palette('--branding-label-text')};
 
 					${until.desktop} {
-						color: ${palette.text.standfirst};
+						color: ${palette('--standfirst-text')};
 					}
 
 					a {
-						color: ${neutral[20]};
+						color: ${palette('--branding-label-text')};
```

Comment:
Would this not be more semantic, here?

```suggestion
						color: inherit;
```